.github/workflows/publish-packages-1.0.yml

```diff
@@ -32,6 +32,11 @@ jobs:
         fetch-depth: 0 # fetching all
         ref: ${{ matrix.branches }}
 
+    - uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: '6.0.x'
+        include-prerelease: true
```

Comment:
Why do we need `include-prerelease`?

Reply:
copied from existing workflows.
The whole step would be removed with https://github.com/open-telemetry/opentelemetry-dotnet/issues/2563, once GHA hosts pre-install .NET 6.0

Reply:
Got it, thanks.